lib/routes/sms.js

```diff
@@ -98,6 +101,53 @@ module.exports = (log, isA, error, config, customs, sms) => {
           return {}
         }
       }
+    },
+    {
+      method: 'GET',
+      path: '/sms/status',
```

Comment:
Can you document this endpoint in [API.md](https://github.com/mozilla/fxa-auth-server/blob/master/docs/api.md)?